coapclient.py

```diff
@@ -143,7 +143,10 @@ def main():  # pragma: no cover
         usage()
         sys.exit(2)
 
-    host, port, path = parse_uri(path)
+    # host, port, path = parse_uri(path)
```

Comment:
same here... not hard coded

Reply:
Yeah I did make a mess when exploring the library... These scripts aren't used in our tests anyway, so I'll change everything back.